easybuild/easyconfigs/x/xtb/xtb-6.2.3-foss-2019b.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'MesonNinja'
+
+name = 'xtb'
```

Comment:
Please, change name capitalization here and in the file name to match official name. 
```suggestion
name = 'xTB'
```

Reply:
Please, the name of the program is still `xtb`.

Reply:
No problem, the name can stay as `xtb`. We already have older easyconfigs for it with that name as well.